tox.ini

```diff
@@ -4,10 +4,10 @@ filterwarnings =
   ignore::pytest.RemovedInPytest4Warning
 
 [tox]
-envlist = py{27,35,36,37}-django111, py37-flake8
+envlist = py{35,36,37}-django111, py37-flake8
 
 [flake8]
-exclude = *migrations/*
+exclude = *migrations/*, ^subjects/*.py, ^tests/*.py
```

Comment:
Why are these being excluded?

Reply:
I added them to exclude .pyc  files from previous run. I'll have them removed.